tests/e2e/cucumber/features/smoke/copyMove.feature

```diff
@@ -56,4 +66,40 @@ Feature: Copy
     And "Alice" moves the following resource using drag-drop-breadcrumb
       | resource | to          |
       | Sub2     | PARENTCopy4 |
+
+    And "Alice" opens the "files" app
+    And "Alice" opens folder "PARENT"
+    And "Alice" copies the following resources to "PARENT/Sub1" at once using dropdown-menu
+      | resource        |
+      | fileToCopy1.txt |
+      | fileToCopy2.txt |
+    And "Alice" copies the following resources to "PARENT/Sub1" at once using batch-action
```

Comment:
is it different from step `copies the following resource using batch-action`

Reply:
yeah, the step above is for copying/moving single resource whereas, the new step is for copying/moving multiple resource at once using the copy button from batch action bar.
![image](https://github.com/owncloud/web/assets/83579989/a5c1ac03-1a7b-4387-a123-6d900bade146)